acid/templates/partials/status_list.html

```diff
@@ -9,7 +9,7 @@ <h4 class="mt-2 mb-2">Queue: {{ queue.name }}</h4>
                             {{ buildset.name }}
                         </button>
                         Zuul ref: {{ buildset.ref }}
-                        <a href="{{ buildset.review_url }}">{% if buildset.buildset_id %}ID: {{ buildset.buildset_id[:-2] }}{% endif %}</a>
+                        <a href="{{ buildset.review_url }}">{% if buildset.buildset_id %}ID: {{ buildset.buildset_id }}{% endif %}</a>
```

Comment:
If should be before ``<a>``, don't leave link without title. So, better:
```jinja
{% if .... %}
    <a >... </a>
{% endif %}```

Reply:
Agree.